contributors/design-proposals/cloudprovider-storage-metrics.md

```diff
@@ -58,28 +58,39 @@ For GCE we simply use `gensupport.RegisterHook()` to register a function which w
 when request is made and response returns.
 
 To begin with we will start emitting following metrics for GCE. Because these metrics are of type
-`Summary` - both count and latency will be automatically calculated.
+`Histogram` - both count and latency will be automatically calculated.
 
-1. gce_instance_list
-2. gce_disk_insert
-3. gce_disk_delete
-4. gce_attach_disk
-5. gce_detach_disk
-6. gce_list_disk
+All gce metrics will be named - `gce_api_latency_duration_seconds`. Namespace and api request
```

Comment:
Isn't `latency_duration` kind of doubling down on the meaning? Seems either of them would be enough but `request` should be in there somehow. 

How about `gce_api_request_duration_seconds`?